packages/botonic-plugin-flow-builder/src/api.ts

```diff
@@ -30,19 +31,33 @@ export class FlowBuilderApi {
     const newApi = new FlowBuilderApi()
 
     newApi.url = options.url
+    newApi.flowUrl = options.flowUrl
     newApi.flow = options.flow ?? (await newApi.getFlow(options.accessToken))
     newApi.request = options.request
 
+    if (process.env.NODE_ENV === ProcessEnvNodeEnvs.DEVELOPMENT) {
+      await newApi.updateSessionWithUserInfo(options.accessToken)
+    }
+
     return newApi
   }
 
   private async getFlow(token: string): Promise<HtFlowBuilderData> {
-    const { data } = await axios.get(this.url, {
+    const { data } = await axios.get(this.flowUrl, {
       headers: { Authorization: `Bearer ${token}` },
     })
     return data as HtFlowBuilderData
   }
 
+  private async updateSessionWithUserInfo(token: string) {
+    const url = `${this.url}/get-user-info`
+    const response = await axios.get(url, {
+      headers: { Authorization: `Bearer ${token}` },
+    })
```

Comment:
I know it will only be executed in development env, but I recommend to encapsulate calls in a try catch block to avoid the bot "crashing".

Reply:
I have it in mind for a next refactoring, but if this call fails, the one to get the flow will also fail and the bot will have no content to respond